### PR TITLE
Review/cleanup

### PR DESCRIPTION
**Pull request by dbelcham**
Part of the regularly scheduled review/cleanup of docs. https://github.com/Particular/PlatformDevelopment/issues/1700

@DavidBoike can you take a look and make sure I didn't eliminate any important info

### CODE REVIEW

File: samples/nhibernate/custom-mappings/sample.md [hunk 1]
```diff
@@ -1,38 +1,33 @@
...
- * Create a `*.hbm.xml` file for the saga data class
- * Use Fluent NHibernate (popular separate fluent API)
- * Use NHibernate.Mapping.Attributes
+ * Creating a `*.hbm.xml` file for the saga data class
+ * Fluent NHibernate (popular separate fluent API)
+ * NHibernate.Mapping.Attributes
  * Loquacious Configuration (native fluent API)
 
-There are probably even other options, but these are the most frequently used.
+There are other options, but these are the most frequently used. The performance difference between these options can hae small variances, but those will occur during endpoint startup, not when the endpoint is processing messages.
```
Comment (DavidBoike) [inline samples/nhibernate/custom-mappings/sample.md]: hae > have

File: samples/nhibernate/custom-mappings/sample.md [hunk 2]
```diff
@@ -1,38 +1,33 @@
...
 
-NOTE: It is not required to create a custom mapping for all sagas if only one saga needs one. The NHibernate configuration in each sample type demonstrate how both generated and custom mappings can work simultaneously.
-
-NOTE: When creating a custom mapping then mapping the **primary key** and **unique indexes** must be done as part of that mapping.
-
-When using custom mappings ensure unique indexes for all mapped saga properties are correctly mapped. Especially when using optimistic concurrency, a transaction isolation level different from serializable and allow concurrent processing of messages.
-
-Not adding a unique constraint can result in duplicate saga entities as the second insert will **not** fail when inserting the same value when multiple messages are processed concurrently referencing the same saga instance.
+NOTE: When creating a custom mapping, the mapping of the **primary key** and **unique indexes** must be done included.
```
Comment (DavidBoike) [inline samples/nhibernate/custom-mappings/sample.md]: "done included"?

File: samples/nhibernate/custom-mappings/sample.md [hunk 3]
```diff
@@ -1,38 +1,33 @@
...
+ * Creating a `*.hbm.xml` file for the saga data class
+ * Fluent NHibernate (popular separate fluent API)
+ * NHibernate.Mapping.Attributes
  * Loquacious Configuration (native fluent API)
 
-There are probably even other options, but these are the most frequently used.
+There are other options, but these are the most frequently used. The performance difference between these options can hae small variances, but those will occur during endpoint startup, not when the endpoint is processing messages.
 
-Which method to use depends on if mapping as xml, fluent code or code attributes. There can be small performance differences during startup but not when processing messages.
+NOTE: There is no requirement to create a custom mapping for all sagas if a custom mapping is needed by only one saga. The NHibernate configuration in each sample type demonstrate how both generated and custom mappings can work simultaneously.
```
Comment (DavidBoike) [inline samples/nhibernate/custom-mappings/sample.md]: "in each sample type demonstrate how..." ?

File: samples/nhibernate/custom-mappings/sample.md (hunk 2 above)
Comment (dbelcham) [reply samples/nhibernate/custom-mappings/sample.md]: would "dun included" be more Minnesotan?

File: samples/nhibernate/custom-mappings/sample.md (hunk 3 above)
Comment (dbelcham) [reply samples/nhibernate/custom-mappings/sample.md]: The 2nd sentence doesn't really add anything so I removed it completely

**Pull request merged**